src/transform.js

```diff
@@ -97,21 +105,31 @@ export default class i18nTransform extends Transform {
       this.entries = this.entries.sort((a, b) => a.key.localeCompare(b.key))
     }
 
-    this.entries.forEach(entry => {
-      catalog = dotPathToHash(
+    let uniqueCount = this.entries.length
+    for (const entry of this.entries) {
+      const { duplicate, conflict } = dotPathToHash(
         entry,
         catalog,
         {
           separator: this.options.keySeparator,
           value: this.options.defaultValue
         }
       )
-    })
+      if (duplicate) {
+        uniqueCount -= 1
+        if (conflict) {
+          warn(`Found same keys with different values: ${entry.key}`)
```

Comment:
Could you make this an event via `emit`? And I would only call `warn` if `verbose` is true

Reply:
Yes, I could. Don't you think that this should be logged regardless of the verbosity level? After all, this is sort of a programming error, isn't it?

Reply:
I don't see this as an error, it is a conflict. If we have an event, people can listen for it when they need too. It is more flexible than a log. The use case would be someone using this with Gulp for instance, they might not want to pollute the log but still catch the conflict.